src/zh/manhuaren/src/eu/kanade/tachiyomi/extension/zh/manhuaren/Manhuaren.kt

```diff
@@ -123,20 +149,38 @@ class Manhuaren : HttpSource() {
             .build()
 
         val simSerialNumber = generateSimSerialNumber()
-        val mac = Random.nextUBytes(6)
-            .joinToString(":") { it.toString(16).padStart(2, '0') }
+        // val mac = Random.nextUBytes(6)
```

Comment:
remove commented code

Reply:
I think it's okay to back these chunks of code up.